### ISSUE TITLE
No success in install

### ISSUE THREAD

**Issue by pajachiet** [EDS-APHP/pySyntheticDatasetGenerator#1]
I am interested by your work, to generate synthetic dataset based on json table-schema of the snds
- explanation https://gitlab.com/healthdatahub/schema-snds/blob/master/documentation/Table-Schema.md
- example schema https://gitlab.com/healthdatahub/schema-snds/blob/master/data/tableschema/BENEFICIAIRE/IR_BEN_R.json

I added missing dependencies in `requirements.txt` (cf https://github.com/pajachiet/pySyntheticDatasetGenerator/commit/b9fe1b9ce0defc6994ac842ee08e89f26dc3e783)

However I get the following error 
`ModuleNotFoundError: No module named 'sdgen.config'`

- Could you please help me to install your work ?
- Do you think it would be a good usecase ?

**Comment by parisni** [comment]
hi @pajachiet,

sorry, i just fix, tested and that's should be fine. please tell me if problem, that should work in few seconds.

About the usecase, this project is based on extended liquibase yaml representation witch is fairly similar to table-schema. So I would say with minor modifications the tool should produce some data, and also some documentation from you (transformed) files.

The tool is based on dsfaker which my collegue @Dubrzr  made several years ago, and some improvements would allow to simulate some complex distributions easily.

**Comment by pajachiet** [comment]
Thanks a lot for your help !

I finally manage to install and run everything (almost… I only miss graphviz plots in schemaSpy output).

I had to make some modification to the code https://github.com/pajachiet/pySyntheticDatasetGenerator/commits/master, mostly related to running it from MacOs.

I also had to install `mawk` and `graphviz` dependencies.

As I dived deeper in the code, I am considering to write a specific library - inspired by yours - tailored for TableSchemas and our usecases. I'll keep playing around with your library to make my mind.

I'll keep you updated anyway.

Thanks again !
